packages/sfui/frameworks/nuxt/src/module.ts

```diff
@@ -0,0 +1,65 @@
+import { defineNuxtModule, addComponent, addImportsSources, installModule } from '@nuxt/kit';
+import * as storefrontUi from '@storefront-ui/vue';
+import { tailwindConfig } from '@storefront-ui/vue/tailwind-config';
+
+// Module options TypeScript interface definition
+export interface ModuleOptions {
+  contentPath?: string;
+}
+
+export default defineNuxtModule<ModuleOptions>({
+  meta: {
+    name: 'storefront-ui',
+    configKey: 'storefrontUi',
+    compatibility: {
+      nuxt: '>=3.0.0',
+    },
+  },
+  // Default configuration options of the Nuxt module
+  defaults: {
+    contentPath: './node_modules/@storefront-ui/vue/**/*.mjs',
+  },
+  async setup(options, nuxt) {
+    const { contentPath } = options;
+
+    // @ts-ignore
+    nuxt.options.tailwindcss = {
+      // @ts-ignore
+      ...nuxt.options.tailwindcss,
+      config: {
+        presets: [tailwindConfig],
+        content: [contentPath],
+        // @ts-ignore
+        ...nuxt.options.tailwindcss?.config,
```

Comment:
Comment:
I think we should try to make it working with proper typings 😄 What's the issue here?

Reply:
`defineNuxtModule` is being imported from a hoisted dependency, but the `tailwindcss` key is dynamically generated based on the setup in the `playground`.

Some options would be `nuxt.options` through a validator like Zod to get the typechecks working, cast nuxt.options as any, or maybe look at not hoisting `@nuxt/kit`? 

Not sure would the best approach would be.

Reply:
100a3271b45a8e6ba85992e423284fd05a8a7056